client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -183,21 +211,21 @@ export default React.createClass( {
 
 				<Input label={ this.translate( 'Postal Code', { textOnly } ) } { ...fieldProps( 'postal-code' ) }/>
 
-				<PrivacyProtection
-					cart={ this.props.cart }
-					countriesList= { countriesList }
-					disabled={ formState.isSubmitButtonDisabled( this.state.form ) }
-					fields={ this.state.form }
-					onButtonSelect={ this.handlePrivacyDialogButtonSelect }
-					onDialogClose={ this.handlePrivacyDialogClose }
-					onDialogOpen={ this.handlePrivacyDialogOpen }
-					onDialogSelect={ this.handlePrivacyDialogSelect }
-					isDialogVisible={ this.state.isDialogVisible }
-					productsList={ this.props.productsList } />
+				{ ( abtest( 'privacyCheckbox' ) !== 'checkbox'
+					? this.renderPrivacySection()
+					: this.renderSubmitButton() ) }
 			</div>
 		);
 	},
 
+	handleCheckboxChange() {
+		if ( cartItems.getDomainRegistrationsWithoutPrivacy( this.props.cart ).length > 0 ) {
+			addPrivacyToAllDomains();
+		} else {
+			removePrivacyFromAllDomains();
```

Comment:
If I have 2 domains in my cart these changes will apply to both domains. I guess that's probably the preferred behaviour.